sonar-swift-plugin/src/main/shell/run-sonar-swift.sh

```diff
@@ -346,6 +348,13 @@ if [ "$unittests" = "on" ]; then
     if [[ ! -z "$binaryName" ]]; then
     	slatherCmd+=( --binary-basename "$binaryName")
     fi
+    if [[ ! -z "$otherBinaryNames" ]]; then
```

Comment:
I would prefer to use a single Sonar property, mandatory, which is a list of binaries. This we can unify `binaryName` and `otherBinaryNames` which are only used in Slather at the moment.

Reply:
@gaelfoppolo can you review the changes?